src/app/payment-options/new-payment-option/new-payment-option.component.html

```diff
@@ -1,26 +1,61 @@
 <div class="container root-content-container">
     <div class="row">
         <div *ngIf="hasNoBankAccounts" class="col-md-12 text-center mb-5 mt-3 h6 gray-text">
-            You currenty have no bank accounts registered.
+            You currently have no bank accounts registered.
         </div>
         <div class="col-md-12">
             <div class="explainer-mini text-uppercase">
                 Enter bank account details
             </div>
         </div>
+
         <div class="col-md-12 mt-3">
-            <input class="form-control w-100 p-4 " id="iban-holder" name="" placeholder="IBAN" type="text">
-            <select class="form-control mt-3 p-2  bank-code-picker" data-live-search="true" id="bankcode-holder" name=""
-                    placeholder="SWIFT/BIC (Optional)" type="text">
-                <option *ngFor="let bankCode of bankCodes; let i = index" value="{{ bankCode.swift_code }}">
-                    {{ bankCode.bank }} ({{ bankCode.city }}) : {{ bankCode.swift_code }}
-                </option>
-            </select>
-            <input class="form-control mt-3 p-4" id="alias-holder" name="" placeholder="Bank account alias" type="text">
+            <form [formGroup]="bankAccountForm">
+
+                <input class="form-control w-100 p-4 "
+                       id="iban-holder"
+                       formControlName="iban"
+                       name=""
+                       placeholder="IBAN"
+                       type="text">
```

Comment:
We should define how we're going to format code. @mightymatth told me we should try and
have around 2 HTML attributes per line (length wise, if attributes are shorter you can add more).

So I would format it like this:
```suggestion
                <input class="form-control w-100 p-4 " id="iban-holder"
                           formControlName="iban" placeholder="IBAN"
                           type="text">
```

Btw, if you don't use name attribute, you can remove it

Reply:
Personally, I prefer more this way, because for me it's much cleaner/readable (Maximilian Schwarzmüller also use it in Angular tutorial from which we learn) 😎 , but ok I will format it your way if that is the format we should use

Reply:
Thnx for name attribute, I will remove it